project/Dependencies.scala

```diff
@@ -74,14 +76,26 @@ object Dependencies {
       Nil
     }
 
+  val extractVs = "[a-zA-Z]*(\\d+)\\.(\\d+)\\.(\\d+).*".r
   def sparkHive(v: String) = "org.apache.spark" %% "spark-hive" % v excludeAll(
     ExclusionRule("org.apache.hadoop"),
     ExclusionRule("org.apache.ivy", "ivy"),
     ExclusionRule("javax.servlet", "servlet-api"),
-    ExclusionRule("org.mortbay.jetty", "servlet-api"),
-    // P.S. Need to comment this out for spark 1.5.2 work with Hive+parquet in CDH 5.4.4+ (hadoop 2.6.0)    
-    ExclusionRule("com.twitter", "parquet-hadoop-bundle")
-  ) excludeAll(parquetList:_*)
+    ExclusionRule("org.mortbay.jetty", "servlet-api")
+  ) excludeAll(parquetList:_*) excludeAll(
+    {
+      val sparkVersion = defaultSparkVersion match { case extractVs(v, m, p) =>  (v.toInt, m.toInt, p.toInt)}
+      val hadoopVersion = defaultHadoopVersion match { case extractVs(v, m, p) => (v.toInt, m.toInt, p.toInt)}
+      import scala.math.Ordering.Implicits._
+      if (sparkVersion >= (1, 5, 2) && hadoopVersion >= (2, 6, 0)) {
```

Comment:
this is because parquet was moved from `com.twitter` to `org.apache`, and since `1.5.2` spark uses new parquet package. and it looks somewhere the old packages are still being used.

so I'm not quite sure what exactly should be the condition on `hadoopVersion`, if any.

e.g. http://mvnrepository.com/artifact/org.apache.parquet/parquet-hadoop-bundle/1.7.0